src/components/chart/vegaLiteChart/vegaLiteChart.stories.tsx

```diff
@@ -0,0 +1,164 @@
+import type { Meta, StoryFn } from '@storybook/react'
+import React from 'react'
+
+import VegaLiteChart from './vegaLiteChart'
+const meta: Meta<React.ComponentProps<typeof VegaLiteChart>> = {
+  title: 'Rustic UI/Chart/Vega-Lite Chart',
+  component: VegaLiteChart,
+  tags: ['autodocs'],
+  parameters: {
+    layout: 'centered',
+  },
+}
+
+export default meta
+meta.argTypes = {
+  ...meta.argTypes,
+  theme: {
+    table: {
+      type: {
+        summary: 'Theme object.\n',
+        detail:
+          'A theme object has the following fields:\n' +
+          '  light: A light theme string that is supported by vega-themes.\n' +
+          '  dark: A dark theme string that is supported by vega-themes.\n' +
+          'Refer to the [vega-themes](https://github.com/vega/vega-themes) documentation for more information.',
```

Comment:
This part doesn't seem to be rendering correctly (the link). Maybe you can add it to the prop description instead.

Reply:
I think vega also accepts an object. I dont think we should restrict it to string